recipe/doc/recipe_guide.md

```diff
@@ -0,0 +1,76 @@
+
+## Best Practices
+
+Recipe maintainers are encouraged not to use the metapackages for specifying dependencies, but to instead specify only the minimal subset of CUDA components required for usage of their libraries.
+The `*-dev` variants of the packages all have [`run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) set up such that if your package requires a certain CUDA library at run time, specifying the corresponding `dev` package as a host requirement will result in your package exporting the corresponding non-dev package as a runtime dependency.
+
+## CUDA Enhanced Compatibility
+
+Since CUDA 11, [CUDA has offered a number of increased compatibility guarantees](https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#cuda-compatibility-developer-s-guide).
+The most up to date documentation for these may be found in [the CUDA best practices documentation](https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#cuda-toolkit-versioning).
+Of particular interest from a conda packaging perspective is the binary backwards compatibility made by the CUDA Toolkit (CTK): packages built with newer versions of the CTK will also run with older minor versions (within the same major family) of the CTK installed, assuming that your package has properly guarded any usage of APIs introduced in newer versions with suitable checks.
+This guide assumes that you already know how to write and compile code that supports such behavior.
+If your library is properly configured as such, you will need to do a bit of extra work to ensure that your conda package supports this as well:
+
+- You must [ignore the `run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) of any CUDA packages that your package depends on. Otherwise, the `run_exports` would require the runtime libraries to have versions equal to or greater than the versions used to build the package.
+- You must add explicit runtime dependencies (or `run_constrained` for soft/optional dependencies) that specify the desired version range for the dependencies whose `run_exports` have been ignored. 
+
+As an example, consider that you have built a package that requires `libcublas`:
+
+
+```yaml
+requirements:
+  build:
+    - compiler('cuda')
+  host:
+    - libcublas
+    - cuda-version=12.4
+```
+
+By default, at runtime your library will require having the `libcublas` version corresponding to CUDA 12.4 or newer.
+To make this compatible with all CUDA 12 minor versions 12.0+, you must add the following:
+```yaml
+build:
+  # Ignore run exports in your build section
+  ignore_run_exports_from:
+    - {{ compiler('cuda') }}
+    - libcublas-dev
+
+requirements:
+  run:
+    # Since we’ve ignored the run export, pin manually, but set the min to just "x" since we support any libcublas within the same major release, including older versions
```

Comment:
```suggestion
    # Since we've ignored the run export, we pin manually, but set the min to "x" since we support any libcublas within the same major release, including older versions
```